app/views/system/admin/instance/users/update.js.erb

```diff
@@ -0,0 +1,4 @@
+$('.system-admin-instance-users').parents('.container-fluid:first').prepend('<%= flash_messages %>');
+
+// Show users the flash messages
+window.scrollTo(0, 0);
```

Comment:
@kxmbrian I received a warning from capybara of the window scroll before, is it still there ?